lms/djangoapps/student_account/views.py

```diff
@@ -3,28 +3,35 @@
 import logging
 import json
 import urlparse
+import pytz
 
+from datetime import datetime
```

Comment:
Nitpick: this is part of the standard library and should be in the first block of imports.